src/core/internal/traits.d

```diff
@@ -8,23 +8,22 @@
  */
 module core.internal.traits;
 
-/// taken from std.typetuple.TypeTuple
-template TypeTuple(TList...)
-{
-    alias TypeTuple = TList;
-}
-alias AliasSeq = TypeTuple;
 
-template FieldTypeTuple(T)
+// TODO: deprecate these old names...?
```

Comment:
Can't we just remove them? It's `core.internal` so should be fine.

Reply:
That's a large project.

Reply:
`FieldTypeTuple` is only used in this file. `TypeTuple` is used elsewhere in druntime too, but I can make a follow-up PR.

Reply:
Go for it!